src/content/en/updates/2019/08/twas-quickstart.md

```diff
@@ -0,0 +1,221 @@
+
+### A note on signing keys {: #a-note-on-signing-keys }
+
+**Digital Asset Links** take into account the key that an APK has been signed with and a common
+cause for verification failing is to have the signature wrong.
+When you hit *Run* or *Build APK* in Android Studio, the APK will be created with your developer
+**debug key**, which Android Studio automatically generated for you.
+
+If you deploy your app to the Play Store you’ll hit *Build > Generate Signed APK*, which will use a
+different signature, one that you’ll have created yourself (and protected with a password).
+That means that if your **Digital Asset Links** file specifies your **production key**, verification
+will fail when you build with your **debug key**.
+This also can happen the other way around - if the Digital Asset Links file has your **debug key**
+your TWA will work fine locally, then when you sign it and upload it to Play it will fail to verify.
+
+You can put both your debug and production key in your asset link file (see Adding More Keys below),
+but your debug key is less secure (anyone who gets a copy of the file can use it).
+Finally, if you have your app installed on your device with one key, you can’t install the version
+with the other key - you must uninstall the previous version first.
+
+### Building with debug keys {: #building-with-debug-keys }
+
+Just hit *Run ‘app’* (replace ‘app’ with your module name if you’ve changed it).
+
+### Building with release keys {: #building-with-release-keys }
+
+Hit *Build > Generate Signed APK*, choose APK and on the next page hit “Create New” to create a new
+key.
+There’s more information on creating a key
+[here](https://developer.android.com/studio/publish/app-signing#generate-key).
+Once you created your key, hit next and pick the “release” build variant.
+Make sure you tick both the V1 and the V2 signatures (the Play Store won’t let you upload the APK
+otherwise).
+Hit Finish.
+
+After a few seconds a pop up will appear in the bottom right corner giving you the option to locate
+or analyze the APK (if you miss it, you can press on the *Event Log* in the bottom right).
+Annoyingly you’ll need to use adb manually to install the signed APK with
+`adb install app-release.apk`.
+
+<table>
+  <tr><th>Key</th><th>Debug</th><th>Release</th></tr>
+  <tr>
+    <td>When is it created?</td>
+    <td>Automatically by Android Studio.</td>
+    <td>Manually by you.</td>
+  </tr>
+  <tr>
+    <td>When is it used?</td>
+    <td><ul>
+      <li><i>Run 'app'</i>.</li>
+      <li><i>Debug 'app'</i>.</li>
+      <li><i>Build APK</i>.</li>
+    </ul></td>
+    <td><ul>
+      <li><i>Generate Signed APK</i>.</li>
+      <li>When the app is downloaded from the Play Store.</li>
+    </ul></td>
+  </tr>
+</table>
+
+## Creating your asset link file {: #creating-your-asset-link-file }
+
+Now that you have your app installed (with either the debug or release key) we can go about
+generating the Digital Asset Link file.
+I’ve created the
```

Comment:
This tool looks super cool!! But, we should also link to docs on how to do that manually. Not everyone will use the tool, and since it's not an officially supported tool, we need to have docs so that if it ever goes away... :(